### PR TITLE
[Nu-1882] change parameter parameter list item labels

### PR DESCRIPTION
**Pull request by Dzuming**
## Describe your changes

- Any values with suggestion
image

- Fixed list
image

## Checklist before merge
- [x] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

## Summary by CodeRabbit

- **New Features**
	- Enhanced user feedback for duplicate entries with updated error messages in the fragment modal.
	- Added customizable input labels for user-defined lists in the UI.

- **Bug Fixes**
	- Improved validation and error handling for various input modes in the fragment feature.

- **Documentation**
	- Updated test cases to ensure comprehensive coverage of edge cases and expected behaviors.

### CODE REVIEW

File: designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx [hunk 1]
```diff
@@ -166,7 +168,7 @@ export const UserDefinedListInput = ({
 
     return (
         <FormControl>
-            <SettingLabelStyled>{t("fragment.addListItem", "Add list item:")}</SettingLabelStyled>
+            <SettingLabelStyled>{inputLabel || t("fragment.addListItem", "Add list item:")}</SettingLabelStyled>
```
Comment (arkadius) [inline designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx:171]: Why we still have this fallback? Let's remove it if it is unused or at least change to "List items" to avoid further bugs

Comment (Dzuming) [reply designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx:171]: The intention was to leave this property optional, and not force someone who wants to unit test this component to think about label value, but since we don't have any unit/integration tests (only e2e) i will remove this fallback

Comment (Dzuming) [reply designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx]: Done ✔️

**Review by arkadius — APPROVED** [review]
